pegasus/sites.v3/code.org/views/congrats.haml

```diff
@@ -24,8 +18,8 @@
         %img{:src=>"/images/fit-540/#{certificate_template_for(cert_script)}", :width=>"100%"}
 
     #certificate.col-50
-      = view :congrats_certificate
-      = view :congrats_share, cert_id:cert_id, facebook:facebook, twitter:twitter
+      = view :csf_congrats_certificate, course: course
+      = view :congrats_share, facebook:facebook, twitter:twitter
```

Comment:
It looks like you're replacing congrats_certificate with csf_congrats_certificate for all users on the congrats page. That doesn't seem right. Do you perhaps want to show this view conditionally instead?